src/main/java/uk/gov/justice/digital/hmpps/whereabouts/controllers/CourtController.kt

```diff
@@ -49,4 +50,11 @@ class CourtController(private val courtService: CourtService) {
 
     return VideoLinkAppointmentsResponse(appointments = courtAppointments)
   }
+
+  @PostMapping(consumes = [MediaType.APPLICATION_JSON_VALUE], path = ["/video-link-bookings"])
```

Comment:
Be good to describe what other responses this endpoint could return. There is still a bit of tech dept in some of the other endpoints but I'm working on trying to improve them as I go.